styles/globals.css

```diff
@@ -120,4 +120,10 @@ abbr[title] {
   .DocSearch-Container {
     position: fixed !important;
   }
+  
+  .px-14{
+    padding-left: 0;
+    padding-right: 0;
+  }
```

Comment:
Hey @msaini0r, you can't change the CSS of the Tailwind classes like this. You need to update the CSS of the images using tailwind classes.

Reply:
@akshatnema ok got the problem. Just creating a new pr.